### PR TITLE
Use jvm_args_env.sh to set JVM args

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Brief change log

- Move JVM args to jvm_args_env.sh
- Source env file in api/start.sh to solve miss JAVA_HOME in ci
- Change Master expose port in docker

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by EricGao888 — DISMISSED** [review]

File: docs/docs/en/architecture/configuration.md [hunk 1]
```diff
@@ -86,15 +90,16 @@ The directory structure of DolphinScheduler is as follows:
...
+│   ├── bin
+│   │   └── start.sh                        script to start DolphinScheduler worker-server
+│   │   └── jvm_args_env.sh                 script to set JVM args of DolphinScheduler worker-server
+│   ├── conf
+│   │   ├── application.yaml                configurations of worker-server
+│   │   ├── bootstrap.yaml                  configurations for Spring Cloud bootstrap, mostly you don't need to modify this,
+│   │   ├── common.properties               configurations of common-service like storage, credentials, etc.
+│   │   ├── dolphinscheduler_env.sh         script to load environment variables for worker-server
+│   │   └── logback-spring.xml              configurations of worker-service log
+│         └── libs                                directory of worker-server libs
```
Comment (Radeity) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   │   └── logback-spring.xml              configurations of worker-service log
│   └── libs                                directory of worker-server libs
```

File: docs/docs/zh/architecture/configuration.md [hunk 2]
```diff
@@ -83,16 +88,18 @@ DolphinScheduler的目录结构如下：
...
+│   ├── bin
+│   │   └── start.sh                        DolphinScheduler worker-server 启动脚本
+│   │   └── jvm_args_env.sh                 DolphinScheduler worker-server jvm参数配置脚本
+│   ├── conf
+│   │   ├── application.yaml                worker-server配置文件
+│   │   ├── bootstrap.yaml                  Spring Cloud 启动阶段配置文件, 通常不需要修改
+│   │   ├── common.properties               公共服务（存储等信息）配置文件
+│   │   ├── dolphinscheduler_env.sh         worker-server环境变量配置加载脚本
+│   │   └── logback-spring.xml              worker-service日志配置文件
+│         └── libs                                worker-server依赖jar包存放目录
```
Comment (Radeity) [inline docs/docs/zh/architecture/configuration.md]: Ditto.

File: docs/docs/en/architecture/configuration.md (hunk 1 above)
Comment (ruanwenjun) [reply docs/docs/en/architecture/configuration.md]: Done

File: docs/docs/zh/architecture/configuration.md (hunk 2 above)
Comment (ruanwenjun) [reply docs/docs/zh/architecture/configuration.md]: Done

**Review by Radeity — DISMISSED** [review]
LGTM

**Review by EricGao888 — DISMISSED** [review]

**Review by caishunfeng — APPROVED** [review]
+1

File: dolphinscheduler-master/src/main/docker/Dockerfile [hunk 3]
```diff
@@ -25,6 +25,6 @@ WORKDIR $DOLPHINSCHEDULER_HOME
 
 ADD ./target/master-server $DOLPHINSCHEDULER_HOME
 
-EXPOSE 12345
+EXPOSE 5678 5679
```
Comment (Radeity) [inline dolphinscheduler-master/src/main/docker/Dockerfile:28]: Should remain unchanged?

Comment (ruanwenjun) [reply dolphinscheduler-master/src/main/docker/Dockerfile:28]: The master should expose 5678/5679, 12345 is used by api.

**Pull request merged**